packages/playground/blueprints/src/lib/steps/run-sql.ts

```diff
@@ -0,0 +1,77 @@
+import { StepHandler } from '.';
+import { rm } from './rm';
+
+/**
+ * @inheritDoc runSql
+ * @hasRunnableExample
+ * @example
+ *
+ * <code>
+ * {
+ *		"step": "runSql",
+ *		"sql": {
+ *			"resource": "literal",
+ *			"name": "schema.sql",
+ *			"contents": "SELECT * FROM wp_posts"
+ *		},
+ * }
+ * </code>
+ */
+export interface RunSqlStep<ResourceType> {
+	/**
+	 * The step identifier.
+	 */
+	step: 'runSql';
+	/**
+	 * The SQL to run.
+	 */
+	sql: ResourceType;
+}
+
+/**
+ * Run an SQL file.
+ */
+export const runSql: StepHandler<RunSqlStep<File>> = async (
+	playground,
+	{ sql },
+	progress?
+) => {
+	progress?.tracker.setCaption(`Executing SQL File`);
+
+	await playground.writeFile(
+		'/tmp/___schema-backup___.sql',
```

Comment:
Let's define a const instead of repeating the string literal three times – we'll make it easier to change and reduce any chances of changing the one but not the other.

Also, how about `schema-to-import.sql` as a name instead of `___schema-backup___.sql`?